### PR TITLE
Occupation dropdown backend

### PR DESCRIPTION
**Pull request by hrit2773**
## Proposed Changes

- Changed the enum class

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Missing migrations. Run `python manage.py makemigrations`

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .PHONY: build, re-build, up, down, list, logs, test, makemigrations
 
 
-DOCKER_VERSION := $(shell docker --version 2>/dev/null)
+DOCKER_VERSION := docker --version
```
Comment (rithviknishad) [inline Makefile]: Why was this changed?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by hrit2773** [comment]
@rithviknishad I changed this in my windows machine just to run the command and pushed it by mistake I changed it back and i guess its fine now added migration also

**Review by rithviknishad — APPROVED** [review]

File: care/facility/migrations/0426_alter_patientmetainfo_occupation.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    (2, "BUSINESSMAN"),
+                    (3, "HEALTH_CARE_WORKER"),
+                    (4, "HEALTH_CARE_LAB_WORKER"),
+                    (5, "ANIMAL_HANDLER"),
+                    (6, "HEALTHCARE_PRACTITIONER"),
+                    (7, "PARADEMICS"),
+                    (8, "Business_related"),
+                    (9, "ENGINEER"),
+                    (10, "TEACHER"),
+                    (11, "Other_Professions"),
```
Comment (rithviknishad) [inline care/facility/migrations/0426_alter_patientmetainfo_occupation.py]: Do we need both "Other_Pressions" and "OTHERS"

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: care/facility/migrations/0426_alter_patientmetainfo_occupation.py (hunk 2 above)
Comment (hrit2773) [reply care/facility/migrations/0426_alter_patientmetainfo_occupation.py]: @rithviknishad maybe we can consider other skilled professions that was mentioned in the dropdown suggstions list so i added

Comment (hrit2773) [reply care/facility/migrations/0426_alter_patientmetainfo_occupation.py]: @rithviknishad in the frontend Other professional occupations were mentioned and professional occupation is a separate type and the corresponding value is OTHER_PROFESSIONS not to be confused with OTHERS and its not the same. This is needed.

Comment (rithviknishad) [reply care/facility/migrations/0426_alter_patientmetainfo_occupation.py]: Rename it in backend too. It could lead to confusions.

Also change to use uppercase for all

**Comment by hrit2773** [comment]
@rithviknishad I capitalized the names and also capitalized in the frontend

**Comment by sainak** [comment]
@hrit2773 delete the migrations and rerun makemigrations

**Comment by hrit2773** [comment]
@sainak @rithviknishad plz review this pr

File: care/facility/models/patient.py [hunk 3]
```diff
@@ -570,7 +570,33 @@ class OccupationEnum(enum.Enum):
...
+        CONSTRUCTION_EXTRACTION = 23
+        AGRI_NATURAL = 24
+        PRODUCTION_OCCUPATION = 25
+        PILOT_FLIGHT = 26
+        VEHICLE_DRIVER = 27
+        MILITARY = 28
+        HOMEMAKER = 29
+        OTHERS = 30
+        UNKNOWN = 31
+        NOT_APPLICABLE = 32
```
Comment (rithviknishad) [inline care/facility/models/patient.py]: `OTHERS` was previously `6` and is now `30`. And `6` is now Healthcare Practitioner. This would mean, all previously entered patients with occupation as "Other" before this PR would become a Healthcare Practitioner once this is merged.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by hrit2773** [comment]
@rithviknishad yes you are right OTHERS would be misread as HEALTHCARE PRACTIOTIONER so changed it. And i made changes in the frontend also

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@hrit2773 there is migration issue, fix it

**Comment by hrit2773** [comment]
@nihal467 but it's working fine when I'm running it the occupation is being added and displayed properly. Should I re-run the migrations?

**Comment by hrit2773** [comment]
@nihal467 can you check it now maybe there were some conflicting migrations before

**Review by sainak — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]
